tools/ReleaseTools.psm1

```diff
@@ -332,28 +345,25 @@ function New-DraftRelease {
         [ValidateSet([RepoNames])]
         [string]$RepositoryName
     )
+    param(
+        [Parameter(ValueFromPipeline)]
+        [string[]]$Assets
+    )
     $Version = Get-Version -RepositoryName $RepositoryName
     $Changelog = (Get-FirstChangelog -RepositoryName $RepositoryName) -join "`n"
     $ReleaseParams = @{
         Draft      = $true
+        # NOTE: We rely on GitHub to create the tag.
         Tag        = "v$Version"
+        Committish = "release/v$Version"
         Name       = "v$Version"
         Body       = $ChangeLog
         PreRelease = [bool]$Version.PreReleaseLabel
         # TODO: Pass -WhatIf and -Confirm parameters correctly.
     }
 
-    if ($PSCmdlet.ShouldProcess("$RepositoryName/v$Version", "git tag")) {
```

Comment:
Looks like this command will no longer support shouldprocess, so need to take it out of the cmdletbinding